.github/workflows/csharpier.yml

```
@@ -29,8 +29,11 @@
 jobs:
 
   csharpier:
-    permissions: write-all
     runs-on: ubuntu-latest
+    runs-on: ubuntu-latest
```

actionlint (1.7.4)

key "runs-on" is duplicated in "csharpier" job. previously defined at line:32,col:5

(syntax-check)